sequencer/database/utils.go

```diff
@@ -99,7 +99,7 @@ func InitSQLDB(port int, host, user, password, name string) (*sqlx.DB, error) {
 
 // InitTestSQLDB opens test PostgreSQL database
 func InitTestSQLDB() (*sqlx.DB, error) {
-	err := godotenv.Load("../.env")
+	err := godotenv.Load("../../.env")
```

Comment:
Yeah, I've kept the env variables inside the `sequencer` folder relative path - `sequencer/.env` . For this path on my local it was able to pick the env variables once I changed the path to `../../.env` .

Can you please confirm if this is not the case on your side.

Reply:
@Shailu-s I just realized having a relative `.env` path is not a good idea because apparently go sets the working directory to the test files location. So we'll need to update the .env path depending on the test path location. I just tested this by creating a dummy test in `sequencer/utils_test.go` and loading `"../../.env"` failed. I have two suggestions:
- strongly suggest removing loading `.env` and use `Taskfile.yml`. You can set a specific unit test by adding `-run UnitTestName`
- if we have to keep this logic then we need to dynamically locate `.env` and load it:
```go
cwd, err := os.Getwd()
	if err != nil {
		log.Info("Error getting current working directory")
	}
	parts := strings.Split(cwd, string(os.PathSeparator))
	for len(parts) > 0 && parts[len(parts)-1] != "sequencer" {
		parts = parts[:len(parts)-1]
	}
	var envPath string
	if len(parts) > 0 {
		envPath = filepath.Join(strings.Join(parts, string(os.PathSeparator)), ".env")

		err = godotenv.Load(envPath)
		if err != nil {
			log.Fatalf("Error loading .env file: %v", err)
		}
	} else {
		log.Info("'sequencer' directory not found in the path")
	}
	err = godotenv.Load(envPath)
```